gum/backend-linux/gumprocess-linux.c

```diff
@@ -1656,6 +1656,7 @@ gum_linux_cpu_type_from_pid (pid_t pid,
   guint8 * auxv;
   gsize auxv_size, i;
   GumCpuType cpu32, cpu64;
+  guint64 * auxv_type;
```

Comment:
Let's move this down to the block where it's used.

Reply:
Fixed